api/pom.xml

```diff
@@ -106,6 +106,25 @@
                             </arguments>
                         </configuration>
                     </execution>
+                    <execution>
```

Comment:
We can consolidate this later.  I just added the 2nd execution to get fresh CRD's in the helm chart.